alma/alma-8.x/alma-8.6-hpc/install_mpis.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -ex
+
+# Load gcc
+GCC_VERSION=gcc-9.2.0
+export PATH=/opt/${GCC_VERSION}/bin:$PATH
+export LD_LIBRARY_PATH=/opt/${GCC_VERSION}/lib64:$LD_LIBRARY_PATH
+set CC=/opt/${GCC_VERSION}/bin/gcc
+set GCC=/opt/${GCC_VERSION}/bin/gcc
+
+
+INSTALL_PREFIX=/opt
+
+# HPC-X v2.11
+HPCX_VERSION="v2.11"
+TARBALL="hpcx-${HPCX_VERSION}-gcc-MLNX_OFED_LINUX-5-redhat8-cuda11-gdrcopy2-nccl2.11-x86_64.tbz"
+HPCX_DOWNLOAD_URL=https://azhpcstor.blob.core.windows.net/azhpc-images-store/${TARBALL}
+HPCX_FOLDER=$(basename ${HPCX_DOWNLOAD_URL} .tbz)
+
+$COMMON_DIR/download_and_verify.sh $HPCX_DOWNLOAD_URL "6cf828435caf33981981dd91227761f915f8907e99afd2cbe032422ada6f0a92"
+tar -xvf ${TARBALL}
+mv ${HPCX_FOLDER} ${INSTALL_PREFIX}
+HPCX_PATH=${INSTALL_PREFIX}/${HPCX_FOLDER}
+$COMMON_DIR/write_component_version.sh "HPCX" $HPCX_VERSION
+
+# Setup module files for MPIs
+mkdir -p /usr/share/Modules/modulefiles/mpi/
+
+# HPC-X
+cat << EOF >> /usr/share/Modules/modulefiles/mpi/hpcx-${HPCX_VERSION}
+#%Module 1.0
+#
+#  HPCx ${HPCX_VERSION}
+#
+conflict        mpi
+module load ${HPCX_PATH}/modulefiles/hpcx
+EOF
+
+# Create symlinks for modulefiles
+ln -s /usr/share/Modules/modulefiles/mpi/hpcx-${HPCX_VERSION} /usr/share/Modules/modulefiles/mpi/hpcx
+
+# Install platform independent MPIs
+$ALMA_COMMON_DIR/install_mpis.sh ${GCC_VERSION} ${HPCX_PATH}
+
+# cleanup downloaded tarballs
+rm -rf *.tbz *.tar.gz *offline.sh
```

Comment:
where are these tar.gz, offline.sh files generated?

Reply:
These files are generated from the gcc installation and also from other MPI installations

Reply:
Sorry, let me clarify a bit more. 
I meant if the *.tar.gz and *.offline.sh are downloaded in this script self. If they are downloaded in `$ALMA_COMMON_DIR/install_mpis.sh`, let's remove clean up them inside that script.

Reply:
Separated the cleanup of tarballs and installation folders